### ISSUE TITLE
Invalid CONTAINS generation

### ISSUE THREAD

**Issue by alexfalkowski** [rsim/oracle-enhanced#463]
If you look at 

```
https://github.com/rsim/oracle-enhanced/blob/master/lib/active_record/connection_adapters/oracle_enhanced_context_index.rb
```

And look at module ContextIndexClassMethods, you will see:

```ruby
def contains(column, query, options ={})
  score_label = options[:label].to_i || 1
  where("CONTAINS(#{connection.quote_column_name(column)}, ?, #{score_label}) > 0", query).
  order("SCORE(#{score_label}) DESC")
 end
```

The contains generates a string, which is fine for a single column, however when we join onto multiple tables it generates this:

```sql
SELECT "ANNUAL_DEPARTMENTS".* 
FROM "ANNUAL_DEPARTMENTS" 
INNER JOIN "YEARS" ON "YEARS"."ID" = "ANNUAL_DEPARTMENTS"."YEAR_ID" 
INNER JOIN "ORG_UNITS" ON "ORG_UNITS"."ID" = "ANNUAL_DEPARTMENTS"."ORG_UNIT_ID" 
WHERE (CONTAINS('annual_departments.name', '%T%', 0) > 0) AND (org_units.ancestry_depth = 2) 
ORDER BY SCORE(0) DESC, years.year DESC, annual_departments.name ASC;
```

What you really want it to be is:

```sql
SELECT "ANNUAL_DEPARTMENTS".* 
FROM "ANNUAL_DEPARTMENTS" 
INNER JOIN "YEARS" ON "YEARS"."ID" = "ANNUAL_DEPARTMENTS"."YEAR_ID" 
INNER JOIN "ORG_UNITS" ON "ORG_UNITS"."ID" = "ANNUAL_DEPARTMENTS"."ORG_UNIT_ID" 
WHERE (CONTAINS(annual_departments.name, '%T%', 0) > 0) AND (org_units.ancestry_depth = 2) 
ORDER BY SCORE(0) DESC, years.year DESC, annual_departments.name ASC;
```

Without the quotes.

I am using 1.4.3 version of the gem and the database is Oracle Database 11g 11.2.0.2.0 - 64bit Production

Hopefully this helps.

**Comment by yahonda** [comment]
Thanks for the information. Sorry for my late update. It helps me to tackle #664

**Comment by yahonda** [comment]
Closed as #758 has been merged to master. Thanks for reporting.
